bin/symd.py

```diff
@@ -143,19 +143,19 @@ def get_yang_modules(yfiles, tag):
                     rev = None
                 # IF we already have a module with a lower revision, replace it now
                 try:
-                    en = G.node[name]
+                    en = G.nodes[name]
```

Comment:
Object of type "(data: bool = False) -> list[tuple[Unknown, Unknown]]" is not subscriptable